core/services/ocr2/plugins/ocr2keeper/evm21/streams_lookup.go

```diff
@@ -151,11 +151,13 @@ func (r *EvmRegistry) streamsLookup(ctx context.Context, checkResults []ocr2keep
 	var wg sync.WaitGroup
 
 	for i, lookup := range lookups {
-		i := i
 		wg.Add(1)
-		r.threadCtrl.Go(func(ctx context.Context) {
-			r.doLookup(ctx, &wg, lookup, i, checkResults, lggr)
-		})
+		func(i int, lookup *StreamsLookup) {
```

Comment:
Couldn't you just define another local var for lookup?

Reply:
Yes! But I wasn't sure if this was nicer lol